docs/website/docs/index.md

```diff
@@ -64,7 +64,9 @@ like [SPIR-V](https://www.khronos.org/spir/).
 
 ## Workflow overview
 
-Using IREE involves these general steps:
+Specific examples outlining IREE's workflow can be found in the
+[User Getting Started Guide](./getting-started/index.md). Using IREE involves four
+general steps:
 
 1. **Import your model**
```

Comment:
there's a link to ./ml-frameworks a few lines down from here - can you redirect that and any others?

Reply:
Should be fixed up.